core/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/ipfs-rust/rust-ipld"
 default = ["std"]
 std = ["anyhow/std", "cid/std", "multibase/std", "multihash/std", "thiserror"]
 serde-codec = ["cid/serde-codec", "serde"]
+arb = ["quickcheck", "getrandom/js"]
```

Comment:
You should be able to remove the `"getrandom/js"`, this should make things compile in `no_std` environments.

Reply:
Removed